pandas/tests/plotting/test_series.py

```diff
@@ -270,14 +270,30 @@ def test_bar_ignore_index(self):
 
     def test_bar_user_colors(self):
         s = Series([1, 2, 3, 4])
+
+        expected = [
+            (1.0, 0.0, 0.0, 1.0),
+            (0.0, 0.0, 1.0, 1.0),
+            (0.0, 0.0, 1.0, 1.0),
+            (1.0, 0.0, 0.0, 1.0),
+        ]
+
         ax = s.plot.bar(color=["red", "blue", "blue", "red"])
         result = [p.get_facecolor() for p in ax.patches]
+        assert result == expected
+
+    def test_bar_user_colors_c(self):
+        s = Series([1, 2, 3, 4])
+
         expected = [
             (1.0, 0.0, 0.0, 1.0),
             (0.0, 0.0, 1.0, 1.0),
             (0.0, 0.0, 1.0, 1.0),
             (1.0, 0.0, 0.0, 1.0),
         ]
+
+        ax = s.plot.bar(c=["red", "blue", "blue", "red"])
+        result = [p.get_facecolor() for p in ax.patches]
```

Comment:
```suggestion
        ax = s.plot.bar(**{kw: ["red", "blue", "blue", "red"]})
```
parametrize the test, dont duplicate it

Reply:
Thanks a lot for teaching how to parametrize a test.